packages/core/bootstrap/src/lib/modules/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './requester'
 export * from './logger'
 export * from './selector'
 export * from './validator'
+export * from './Overrider'
```

Comment:
Let's keep the naming style the same as the others
```suggestion
export * from './overrider'
```